### PR TITLE
[docs] Small fixes on the release docs

### PR DESCRIPTION
**Pull request by dpgaspar**
### CATEGORY

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [X] Documentation

### SUMMARY
Small fixes on the release docs, around RC version and `flask fab` cli

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by mistercrunch — APPROVED** [review]

File: RELEASING/README.md [hunk 1]
```diff
@@ -211,6 +203,14 @@ Then tag the final release:
...
 proper template
 
+### Publishing a Convenience Release to PyPI
+From the root of the repo running ./pypi_push.sh will build the
+Javascript bundle and echo the twine command allowing you to publish
+to PyPI. You may need to ask a fellow committer to grant
+you access to it if you don't have access already. Make sure to create
+an account first if you don't have one, and reference your username
+while requesting access to push packages.
+
```
Comment (villebro) [inline RELEASING/README.md]: Unrelated, but the `RELEASING` directory might be a more appropriate location for the `pypy_push.sh` script.

**Review by villebro — APPROVED** [review]
One small non-blocking comment, LGTM

File: RELEASING/README.md (hunk 1 above)
Comment (mistercrunch) [reply RELEASING/README.md]: right or even `scripts/`, though afaic feel free to merge as is and do a follow up PR or issue

Comment (dpgaspar) [reply RELEASING/README.md]: Agree, the release docs and scripts are improving, still would like to fully (or almost) automating this process

**Pull request merged**